### ISSUE TITLE
Error after installing Documentation Plugins

### ISSUE THREAD

**Issue by nythyatdora** [strapi/strapi#7029]
**Bug**
Currently, I am using Strapi v3.0.5 on Docker Container running locally. After installing Documentation Plugin, I was unable to start the server. 

I tried to create new project using the lastest image, and it worked fine. But after I installed the Documentation plugin, it shows the same error as below.

**Code snippets**
```

Starting your app...

[2020-07-16T08:33:51.843Z] debug ⛔️ Server wasn't able to start properly.

[2020-07-16T08:33:51.845Z] error Error: 'darwin-x64' binaries cannot be used on the 'linux-x64' platform. Please remove the 'node_modules/sharp/vendor' directory and run 'npm install'.

at Object.hasVendoredLibvips (/srv/app/node_modules/sharp/lib/libvips.js:67:13)

at Object.<anonymous> (/srv/app/node_modules/sharp/lib/constructor.js:7:22)

at Module._compile (internal/modules/cjs/loader.js:1138:30)

at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)

at Module.load (internal/modules/cjs/loader.js:986:32)

at Function.Module._load (internal/modules/cjs/loader.js:879:14)

at Module.require (internal/modules/cjs/loader.js:1026:19)

at require (internal/modules/cjs/helpers.js:72:18)

at Object.<anonymous> (/srv/app/node_modules/sharp/lib/index.js:3:15)

at Module._compile (internal/modules/cjs/loader.js:1138:30)
```

**System**
- Node version: v14.5.0
- NPM version: v6.14.6
- Strapi version: v3.0.5
- Database: MongoDB
- Operating system: macOS Catalina v10.15.5

**Comment by Ballrock** [comment]
Not sur if it will be useful for someone but for those wondering, i got a similar problem while updating 3.1.0 to 3.2.1.
I was too executing Strapi in Docker on OSX (but since the nature of the problem i presume it can happen on Windows too)

My error was to execute `yarn install` in host the environment. 

The `sharp` dependency installing according to the OS that execute the yarn/npm command, in my case it was on the host so yarn downloaded __darwin-x64__ executables in node_modules folder.
Strapi was executed on a Docker environment with the [official image](https://github.com/strapi/strapi-docker) (that depends on `alpine`), so i got this error because the docker environment want __linux-x64__ executable. 
It's rather logical since the /app folder was mounted by the `volumes` directive in my docker-compose, sharing like this the host's "darwin-x64" node_modules : 
```
volumes:
      - ./app:/srv/app
```

But it's in fact only a "dev env" error.

In conclusion when you need to update Strapi or it's dependencies, do not use `yarn` directly in the host. Just delete node_modules and build folder, the `entrypoint` (https://github.com/strapi/strapi-docker/blob/master/strapi/docker-entrypoint.sh) will automatically redownload/rebuild those 😉

**Comment by derrickmehaffy** [comment]
@Ballrock can you make a PR to the strapi-docker repo and fix that? You came across a very good point (sharp has specific pre-built versions of the libvips library)
